MMPA compile error with Microsoft Visual C++ Compiler for Python 2.7

when building the current tip of the master branch (a783197) on windows using the Microsoft Visual C++ Compiler for Python 2.7, the following error is issued:

C:\Users\riccardo\Anaconda\conda-bld\work\Code\GraphMol\MMPA\MMPA.cpp(206) : error C2039: 'at' : is not a member of 'std::map<_Kty,_Ty>'
        with
        [
            _Kty=unsigned int,
            _Ty=unsigned int
        ]

the 'at' method of std::map is a C++11 feature and I'm afraid it's not supported by msvc 2008.

--- a/Code/GraphMol/MMPA/MMPA.cpp
+++ b/Code/GraphMol/MMPA/MMPA.cpp
@@ -203,8 +203,8 @@ std::cout<<"isotope>=3: invalid fragments. fragment with maxCut connection point
                             || newAtomMap.end() == newAtomMap.find(bond->getEndAtomIdx())
                             || visitedBonds.end() != visitedBonds.find(bond->getIdx()) )
                                 continue;
-                            unsigned ai1 = newAtomMap.at(bond->getBeginAtomIdx());
-                            unsigned ai2 = newAtomMap.at(bond->getEndAtomIdx());
+                            unsigned ai1 = newAtomMap[bond->getBeginAtomIdx()];
+                            unsigned ai2 = newAtomMap[bond->getEndAtomIdx()];
                             unsigned bi  = side_chains->addBond(ai1, ai2, bond->getBondType());
                             visitedBonds[bond->getIdx()] = bi;
                         }
@@ -240,8 +240,8 @@ if(iCore != -1)
                         || newAtomMap.end() == newAtomMap.find(bond->getEndAtomIdx())
                         || visitedBonds.end() != visitedBonds.find(bond->getIdx()) )
                             continue;
-                        unsigned ai1 = newAtomMap.at(bond->getBeginAtomIdx());
-                        unsigned ai2 = newAtomMap.at(bond->getEndAtomIdx());
+                        unsigned ai1 = newAtomMap[bond->getBeginAtomIdx()];
+                        unsigned ai2 = newAtomMap[bond->getEndAtomIdx()];
                         unsigned bi  = core->addBond(ai1, ai2, bond->getBondType());
                         visitedBonds[bond->getIdx()] = bi;
                     }

